codegen/winterfell/src/air/mod.rs

```diff
@@ -108,7 +108,14 @@ fn add_fn_new(impl_ref: &mut Impl, ir: &AirIR) {
     ));
 
     // define the transition constraint degrees of the aux trace `aux_degrees`.
-    new.line("let aux_degrees = Vec::new();");
+    let mut aux_degrees: Vec<String> = Vec::new();
+    for degree in ir.constraint_degrees(1).iter() {
+        aux_degrees.push(degree.to_string(false));
+    }
```

Comment:
stylistic nit: this could be done in a more idiomatic way as follows:
```Rust
let aux_degrees = ir
    .constraint_degrees(1)
    .iter()
    .map(|degree| degree.to_string(false))
    .collect::<Vec<_>>();
```

Reply:
Fixed!